tests/server/test_databaseservice.py

```diff
@@ -22,27 +22,28 @@
 import pytest
 from asyncpg import Connection
 
-from inmanta import data
+from inmanta import const, data
+from inmanta.server import SLICE_AGENT_MANAGER
 from inmanta.server import config as opt
 from inmanta.server.services import databaseservice
 from utils import log_contains, retry_limited
 
 
-@pytest.mark.skip("To be considered in final cleanup")
 async def test_agent_process_cleanup(server, environment, agent_factory):
     opt.agent_processes_to_keep.set("1")
-    a1 = await agent_factory(environment, hostname="host", agent_map=[], agent_names=["agent1"])
-    a2 = await agent_factory(environment, hostname="host", agent_map=[], agent_names=["agent1"])
-    await asyncio.gather(*[a1.stop(), a2.stop()])
+    a1 = await agent_factory(environment)
+    await a1.stop()
+    a2 = await agent_factory(environment)
+    await a2.stop()
```

Comment:
We can only have one scheduler per env, so I make them in sequence instead of parallel